api_spec.yaml

```diff
@@ -3246,9 +3246,497 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
-          
-      
+  
+  /recipe/multitenancy/tenant:
+    put:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Creates or updates a tenant.
+
+        If `tenantId` is not specified, `defaultTenantId` will be used.
+
+        If `tenantId` is specified and is not equal to `defaultTenantId`, SuperTokens subscription license key is required.
+
+        The `emailpasswordEnabled`, `thirdpartyEnabled`, `passwordlessEnabled` will be treated as `false` by default, if not specified in the request body.
```

Comment:
the values of these will be unchanged if the tenant already exists and if they are not specified in the request body